data/model/gc.py

```diff
@@ -376,6 +418,16 @@ def _purge_pre_oci_tag(tag, context, allow_non_expired=False):
         reloaded_tag.delete_instance()
 
 
+def _purge_uploaded_blob(uploaded_blob, context, allow_non_expired=False):
```

Comment:
I know this is a "private" function and the name is relatively explanatory, but can we add a docstring describing its expected behavior? e.g.

```
Deletes a blob from every persistent storage
```

Reply:
It deletes an UploadedBlob entry, actually